### DISCUSSION TITLE
Etsy store: retrieve API data for ROAS, number of orders and number of views per product ?

### DISCUSSION THREAD

**Discussion by MaximeBournez** [etsy/open-api#687] [Ideas]
Hello, 

I have access to some of my private data by API on my Etsy store. However, the following data seems more complicated to retrieve via API:
⁃ The ROAS per product, over a given period of time
⁃ The number of visits per product, over a given time period
⁃ And the number of orders per product, over a given period of time.

Do you think it is possible to retrieve this data by API? It would help us a lot to have a clearer visualization of the performance of the products we sell. In particular, we would like to be able to export this data in order to have a clear overview of the performance of our products, and thus make better marketing decisions for example.

Thank you very much for your help. 

Maxime

**Comment by benjymouse** [comment]
+1 this is a great suggestion

**Comment by etsyachristensen** [comment]
I have recategorized this into the Ideas list so that it can be more clearly tracked as a suggestion for new features.

We just added the listing view count to the other day to the listings, though this is all-time views, not just over a given period.
